backend/docker-compose.aws.yml

```diff
@@ -13,8 +13,8 @@ services:
       options:
         awslogs-region: ${REGION}
         awslogs-group: ${LOG_GROUP}
+        awslogs-multiline-pattern: "^\\{(?:[^{}]|\\{(?:[^{}]|\\{[^{}]*\\})*\\})*\\}$$"
```

Comment:
This is a scary-enough regex that we'll need a comment block to explain it.

Key points to address:
* What does this regex do?
* What are our assumptions about the formatting of the logs?
* An example of what data this regex is matching.
* Any known limitations of the regex?  What log patterns might not be detected or cause issues that we would need to work around?